docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
It seems we have a winner :+1:
I'm glad of this result. It would have been pretty awkward than `@ngrx` and `redux-observable` use a different name.
I'm very impatient to use it :hatched_chick: